lib/request.js

```diff
@@ -593,9 +593,9 @@ module.exports = {
     switch (field = field.toLowerCase()) {
       case 'referer':
       case 'referrer':
-        return req.headers.referrer || req.headers.referer || '';
+        return req.headers.referrer || req.headers.referer || undefined;
       default:
-        return req.headers[field] || '';
+        return req.headers[field] || undefined;
     }
```

Comment:
Wouldn't this return `undefined` anyway if the `|| undefined` were omitted? Of course nothing wrong with being explicit.